src/main/groovy/io/saagie/plugin/dataops/models/EnvVariable.groovy

```diff
@@ -0,0 +1,11 @@
+package io.saagie.plugin.dataops.models
+
+class EnvVariable {
+	
+	String scope
+	
+	ArrayList<String> name
+	
+	boolean include_all_var = false
```

Comment:
is it the standard way to name variables? with snake_case ?

Reply:
Yes because those are the names that will be used inside the build, and the client request them.

Reply:
Like shown in the image below

![image](https://user-images.githubusercontent.com/57661415/90026148-dab5f700-dcae-11ea-8d25-325027a6687b.png)